toolchain/build-xenon-toolchain

```diff
@@ -302,6 +302,22 @@ function freetype_install
 	echo -e "Done"
 }
 
+# pass the repo name to this function
+function sdl_install
+{
+	echo -e "Cloning $1... "
+	git clone https://github.com/ITotalJustice/$1.git  >> $LOGFILE 2>&1 || fail_with_info
```

Comment:
For this PR to be approved, this needs to not point to your personal repo.

Could you update it to not be your personal repo?

Reply:
Sure, I'll have a look at this pr again when I'm free. Not sure why I linked my own repo